hooks/image/70-flatpak-manifest

```diff
@@ -103,37 +61,90 @@ for runtime in runtimes:
         'version': version,
     }
 
-apps = system.list_installed_refs_by_kind(Flatpak.RefKind.APP)
-for app in apps:
-    name = app.get_name()
-    arch = app.get_arch()
-    branch = app.get_branch()
-    key = '{}/{}/{}'.format(name, arch, branch)
-    remote = app.get_origin()
-    commit = app.get_commit()
-    size = app.get_installed_size()
-    ref = app.format_ref()
-    date = commit_date_string(repo, commit)
-    version = app.get_appdata_version()
-    metadata = GLib.KeyFile()
-    metadata.load_from_bytes(app.load_metadata(), GLib.KeyFileFlags.NONE)
-    try:
-        runtime = metadata.get_string('Application', 'runtime')
-    except GLib.GError:
-        runtime = None
-
-    data['flatpak']['apps'][key] = {
-        'name': name,
-        'arch': arch,
-        'branch': branch,
-        'remote': remote,
-        'commit': commit,
-        'size': size,
-        'ref': ref,
-        'date': date,
-        'version': version,
-        'runtime': runtime,
+    # TODO: It would be nice to consider EIB_FLATPAK_LOCALES and extract the
+    # name and summary in those locales too. Unfortunately while there is
+    # internal API to do this easily, it is not exposed by libflatpak.
+    #
+    # TODO: these function calls get the name in the current locale. We assume
+    # this is the C locale but it probably isn't.
```

Comment:
Actually `eib-chroot` does set `LANG=C.UTF-8`. I don't know why I didn't find that while searching earlier today. `LANG` has been a flavour of `C` since the initial commit, 60fc24b2b711d07323a64c73309b2d78c87a89c4.